client/src/Main.elm

```diff
@@ -1,4 +1,4 @@
-module Main exposing (..)
+module Main exposing (Board, Cell, CellStatus(..), Model, Msg(..), Row, cellView, init, main, rowView, update, view)
```

Comment:
The elm-format is doing this automatically... and I don't like it 🤔

Reply:
Me neither.  Just ignore it.